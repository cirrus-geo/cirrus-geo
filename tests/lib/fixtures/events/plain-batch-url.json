{
    "Parameters": {
        "url": "s3://payloads/test-payload.json"
    }
}
