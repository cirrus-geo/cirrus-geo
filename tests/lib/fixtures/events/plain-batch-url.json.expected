[
    {
        "url": "s3://payloads/test-payload_out.json"
    }
]
